environment/build.gradle

```diff
@@ -77,13 +77,10 @@ dependencies {
     implementation     project(':cluster-dns-connector')
 
     implementation     group: "org.yaml",                  name: "snakeyaml",                                version: snakeYamlVersion
-    implementation     group: "io.springfox",              name: "springfox-swagger2",                       version: swagger2Version
-    implementation     group: "io.springfox",              name: "springfox-core",                           version: swagger2Version
-    implementation     group: "io.springfox",              name: "springfox-swagger-ui",                     version: swagger2Version
     implementation     group: "com.google.code.gson",      name: "gson",                                     version: gsonVersion
     implementation     group: "dnsjava",                   name: "dnsjava",                                  version: dnsjavaVersion
-    implementation     group: "io.springfox",              name: "springfox-swagger2",                       version: swagger2Version
-    implementation     group: "io.springfox",              name: "springfox-swagger-ui",                     version: swagger2Version
+    implementation     group: "io.springfox",              name: "springfox-core",                           version: swaggerSpringFoxVersion
```

Comment:
Sanity check: is `springfox-core` still needed?

Reply:
I haven't found it as a dependency of `sprinfox-boot-starter` in maven central, so haven't tried to remove it